stdlib/public/Platform/androidndk.modulemap

```diff
@@ -0,0 +1,417 @@
+// The module map for the Android NDK.
+module AndroidNDK [system] [no_undeclared_includes] {
```

Comment:
I think that treating this similar to `Darwin` seems better - `Android` would be more appropriate.